soc/arm/ti_k3/Kconfig

```diff
@@ -9,7 +9,6 @@ config SOC_FAMILY_TI_K3
 	bool
 
 if SOC_FAMILY_TI_K3
-
```

Comment:
?

Reply:
@KarthikL1729 can you please update this?